bin/cli.py

```diff
@@ -16,15 +17,25 @@
 
 @main.command()
 def train(
-    name: str = typer.Option(
+    name : str = typer.Option(
         ..., "-n", "--name", help="Name of experiment to save under."
     ),
+    epochs: int = typer.Option(
+        ..., "-e", "--epochs", help="Number of epochs for training."

```

Comment:
you can include default values for these command line arguments by replacing the ... with the default value.